src/js/itemset.js

```diff
@@ -244,6 +246,12 @@ export default {
         const option = document.createElement( 'option' );
         option.textContent = label;
         option.value = value;
+        /**
+         * #510 Show warning if select_one value has spaces
```

Comment:
Let's actually not do this check for select_one. Sorry for the confusion. I was't quite sure either (technically it's not wrong to have a space in a select_one value), and just saw that ODK Collect seems to also allow this.

Reply:
The test form (v2) is still good as it contains 3 select_multiple questions that prompt erros.

Reply:
P.S. `createOption` is not only for select_one. It is used for both multiple and single selects.

Reply:
code removed